sesh/settings.py

```diff
@@ -81,6 +82,19 @@
 # slack key
 SLACK_TEST_KEY = config.get('api', 'slack_test_key')
 
+# use template twoo
+USE_TEMPLATE_TWO = eval(config.get('templates', 'USE_TEMPLATE_TWO'))
```

Comment:
It would probably best to check if template 2 exists. This way it won't error if it doesn't exist.